tests/system/providers/google/cloud/bigquery/example_bigquery_sensors.py

```diff
@@ -98,11 +96,12 @@
     # [END howto_sensor_bigquery_table_defered]
 
     # [START howto_sensor_async_bigquery_table]
-    check_table_exists_async = BigQueryTableExistenceAsyncSensor(
+    check_table_exists_async = BigQueryTableExistenceSensor(
```

Comment:
There is already the same task, that is called check_table_exists_def which is doing the same. I think there is no need to duplicate tasks and we can just remove this one

Reply:
yes @VladaZakharova I missed to see that. thank you